libs/language-server/src/lib/ast/wrappers/pipeline-wrapper.ts

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  BlockDefinition,
+  CompositeBlocktypeDefinition,
+  PipelineDefinition,
+} from '../generated/ast';
+
+import { AstNodeWrapper } from './ast-node-wrapper';
+import { PipeWrapper, createWrappersFromPipeChain } from './pipe-wrapper';
+
+export class PipelineWrapper
+  implements AstNodeWrapper<PipelineDefinition | CompositeBlocktypeDefinition>
+{
+  public readonly astNode: PipelineDefinition | CompositeBlocktypeDefinition;
+
+  allPipes: PipeWrapper[] = [];
+
+  constructor(
+    pipelineDefinition: PipelineDefinition | CompositeBlocktypeDefinition,
```

Comment:
If a parameter can be of multiple types, I wouldn't name it after only one. I'd suggest `container`, `pipelineContainer` or similar here as well.

Reply:
Decided to go with `pipesContainer` in [c868cca](https://github.com/jvalue/jayvee/pull/491/commits/c868cca3414d172164ca61f4435e61d9f9cbd25e)